ipywidgets_bokeh/src/manager.ts

```diff
@@ -191,17 +190,19 @@ export class WidgetManager extends HTMLManager {
   }
 
   override async _create_comm(target_name: string, model_id: string, data?: any, metadata?: any,
-      buffers?: ArrayBuffer[] | ArrayBufferView[]): Promise<IClassicComm> {
+      buffers?: ArrayBuffer[] | ArrayBufferView[]): Promise<shims.services.Comm> {
     const comm = (() => {
       const key = target_name + model_id
-      const comm = this._comms.get(key)
-      if (comm != null)
-        return comm
-      else {
-        const comm = this.kernel.createComm(target_name, model_id)
+      let comm = this._comms.get(key)
+      if (comm == null) {
+        if (this.kernel.hasComm(model_id)) {
+	  comm = (this.kernel as any)._comms.get(model_id)
```

Comment:
I wasn't able to find a public API for this sadly.

Reply:
It's weird one can check if a comm exists, but not retrieve it. In any case, I improved this a little bit to avoid `any` cast.